### PR TITLE
Add User management page

### PR DESCRIPTION
**Pull request by huynguyencs**
screen shot 2018-02-05 at 2 03 18 pm

User management page with list of users query from DB done.
Working on Searching a specific user (search bar) and Edit(user/station) - what does it do btw?

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -47,4 +47,18 @@ router.post('/registration/:badge', jsonParser, function(req, res) {
   res.redirect('/badgein');
 });
 
+router.get('/userManagement', function(req, res) {
+  console.log(req.body)
+  
+
+  var jsonParser = dbAPI.getUsers('2000-01-01', '3000-01-01').done(function(ret){
```
Comment (danieleynis) [inline routes/index.js]: I don't think you need to catch the return value here. Also instead of `.done()` please use `.then()`

**Comment by huynguyencs** [comment]
@danieleynis: Fix those lines as you suggested.

**Comment by huynguyencs** [comment]
resolved conflicts on index.js

**Comment by danieleynis** [comment]
Looks like there are conflicts and issues with the rebase, let's take a look at it tomorrow at the meeting.

**Pull request merged**